cola/models/main.py

```diff
@@ -467,6 +467,8 @@ def stage_paths(self, paths):
 
```

Comment:
I forgot to mention that it would be good to include some of the details that were mentioned in the pull request comment in the commit message's `Extended description` since the actual commit didn't include any of these details.

I tend to merge PRs via cola or git itself rather than github, so it's better to include those details in the commits rather than in the github comments.